MVP_Clicker/Assets/Project/Scripts/Game/EntryPoint.cs

```diff
@@ -47,11 +48,13 @@ private void Start()
 
         private GameConfigs LoadGameConfigs()
         {
-            var bonusesShopScriptableObject = Resources.Load("Bonuses/BonusesShop");
+            ILoadResourcesService loadSystem = new LoadResourcesService();
```

Comment:
it's strange that your name of interface service but you named field with word system. 
Let's name the field like you named interface
ILoadResourcesService loadResourcesService